network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -14,7 +14,7 @@
   <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center flex-sm-row-reverse homepage-feature">
     <div class="col-sm-6 mb-3 mb-sm-0">
         {% image page.hero_image max-200x200 as image_small %}
-        {% image page.hero_image max-400x400 as image_medium %}
+        {% image page.hero_image max-400x400 jpegquality-40 as image_medium %}
```

Comment:
@patjouk I added the jpegquality setting here and the quality difference is visibly apparent.